flytekit/tools/subprocess.py

```diff
@@ -8,22 +8,15 @@ def check_call(cmd_args, **kwargs):
     if not isinstance(cmd_args, list):
         cmd_args = _schlex.split(cmd_args)
 
-    # Jupyter notebooks hijack I/O and thus we cannot dump directly to stdout.
-    with _tempfile.TemporaryFile() as std_out:
-        with _tempfile.TemporaryFile() as std_err:
-            ret_code = _subprocess.Popen(cmd_args, stdout=std_out, stderr=std_err, **kwargs).wait()
+    ret_code = _subprocess.Popen(cmd_args, **kwargs).wait()
 
-            # Dump sub-process' std out into current std out
-            std_out.seek(0)
-            logging.info("Output of command '{}':\n{}\n".format(cmd_args, std_out.read()))
+    # Dump sub-process' std out into current std out
 
-            if ret_code != 0:
-                std_err.seek(0)
-                err_str = std_err.read()
-                logging.error("Error from command '{}':\n{}\n".format(cmd_args, err_str))
+    if ret_code != 0:
```

Comment:
From the logs, there is no "Output of command". So either wait or the std_out.seek is hanging.